index.js

```diff
@@ -1,16 +1,22 @@
 const copier = {};
 module.exports = copier;
 
-const _ = require('lodash');
+const get = require('lodash.get');
 require('colors');
 const fs = require('fs-extra');
 const path = require('path');
 const NODE_MODULES = 'node_modules';
 
-copier.execute = (projectPath, targetPath) => {
-	if (_.isNil(projectPath)) {
+/**
+ * @param {string} projectPath absolute filepath for project root directory
+ * @param {string} targetPath absolute filepath for target directory to copy node_modules into
+ * @returns {Promise<void>}
+ */
+copier.execute = async (projectPath, targetPath) => {
+	if (projectPath === null) {
```

Comment:
_.isNil checks for undefined or null.   If you change this to `projectPath == null` I think it would be equivalent